config/config.go

```diff
@@ -6,6 +6,20 @@ import (
 	"time"
 )
 
+var (
+	DefaultTendermintDir = ".tendermint"
+	DefaultConfigDir     = "config"
+	DefaultDataDir       = "data"
+
+	DefaultConfigFileName  = "config.toml"
```

Comment:
I like these, but we already have the config https://github.com/tendermint/tendermint/blob/develop/config/config.go#L69, so we probably use those

Reply:
i'll tackle that in #705 when diving deeper into the config

Reply:
Mmm I think Anton is right. Best if we use the values in the config object where we need them rather than these package level globals - it should be part of this PR

Reply:
Well - I like having all the strings at the top in one place, but I don't think they should be exported.

The only one that should be exported is `DefaultTendermintDir`